src/components/Header.vue

```diff
@@ -1,40 +1,68 @@
 <script lang="ts">
     import { defineComponent } from 'vue';
+    import CartFlyout from './CartFlyout.vue';
+    import { CartItemInterface } from '../types';
 
     export default defineComponent({
-    name: 'Header',
-    props: {
-        mobileMenuOpen: Boolean,
-        cartOpen: Boolean,
-        totalQuantity: {
-            type: Number,
-            default: 0,
-        },
-    },
+      components: {CartFlyout},
+      name: 'Header',
+      props: {
+          mobileMenuOpen: Boolean,
```

Comment:
In the template, the `toggle-cart` and `toggle-mobile-menu` events are emitted, but in the props definition, the corresponding properties are named `cartOpen` and `mobileMenuOpen`. It would be better to use consistent naming conventions for better readability and maintainability.